src/metabase/util.clj

```diff
@@ -811,7 +811,7 @@
   [message start-time]
   ;; indent the message according to [[*profile-level*]] and add a little down-left arrow so it (hopefully) points to
   ;; the parent form
-  (println (format-color (case (mod *profile-level* 4)
+  (println (format-color (case (int (mod *profile-level* 4))
```

Comment:
small thing but i don't understand the `int` here. I did some simple testing and there was little to no difference as far as i can tell
```clojure
annotate=> (time
             (dotimes [_ 1000000]
               (case (mod 244224 4)
                 0 :zero
                 1 :one
                 2 :two
                 3 :three)))
"Elapsed time: 24.125041 msecs"
nil
annotate=> (time
             (dotimes [_ 1000000]
               (case (int (mod 244224 4))
                 0 :zero
                 1 :one
                 2 :two
                 3 :three)))
"Elapsed time: 33.01825 msecs"
nil
```

Reply:
the only difference is it complains if `*warn-on-reflection*` is enabled since `case` doesn't have a primitive test case or something like that

Reply:
oh so it does. oddly it doesn't seem to impact the performance at all. whatever.

Reply:
Yeah not 100% sure why here, I guess it's because it doesn't know that `*profile-level*` is an `Integer`. I didn't think about it that much TBH